db/default/users.rb

```diff
@@ -58,10 +58,13 @@ def create_admin_user
     admin = Spree::User.new(attributes)
     admin.skip_confirmation!
     admin.skip_confirmation_notification!
+
+    # The default domain example.com is not resolved by all nameservers.
+    ValidEmail2::Address.define_method(:valid_mx?) { true }
```

Comment:
The alternative would be a new attribute on the User model to skip mx domain validation. But I decided to avoid it as long as we don't have other use cases because it would be more code in a much used class.